### ISSUE TITLE
Cannot build on top of mined stone

### ISSUE THREAD

**Issue by abledbody** [jsettlers/settlers-remake#219]
Stone that has been completely mined out cannot have buildings place on top of them, like in the original game.

**Comment by andreas-eberle** [comment]
Can you add a screenshot showing this behavour? I wasn't able to repdoduce it.

**Comment by abledbody** [comment]
@andreas-eberle ![Screenshot](http://puu.sh/l0Bg0/331b529c84.png)
It's possible this is either exclusive to map editor placed stone, or stonecutters can mine stone rubble.

**Comment by michaelzangl** [comment]
I am more concerned that you can build on the tower in that screenshot.

**Comment by andreas-eberle** [comment]
@abledbody: Yes, it seems to be an issue with the stones placed by the map editor. I will check that.
@michaelzangl: What do you mean? You can't build a tower on top of the other tower, but behind it...

**Comment by michaelzangl** [comment]
Yes, the right tower. My bad, I'll blame the one that implemented the preview function.

**Comment by andreas-eberle** [comment]
Yes, this was fixed. Thanks for closing the issue.
